### ISSUE TITLE
Simulation Mode Off not working

### ISSUE THREAD

**Issue by tbaragao** [netzbegruenung/keycloak-mfa-plugins#102]
Hey guys, 

I've tried to use 25.0.1 and 25.0.2 releases, but when set the Simulation Mode Off for sms authentication it's still sending the code in the log as Simulation mode.

**Comment by tbaragao** [comment]
Returning here to provide more information.

In Authentications, I have two flows using SMS, Browser, and Post Login Identity Providers.

I noticed that updating one flow also changed the other, yet when sending the SMS, the simulation mode was still active.

I decided to check directly in the database and found 16 replicas of the same configuration. Only one of these configurations was being updated when making changes, but possibly, another configuration was being used when sending the SMS.

I updated everything in the database, and it started working again. Then, I removed 15 configurations from the database, and everything is functioning normally now, but I'm unsure how these replicas were created.
